ci/test_wheel_dask_cudf.sh

```diff
@@ -16,6 +16,12 @@ rapids-logger "Install dask_cudf, cudf, pylibcudf, and test requirements"
 # generate constraints (possibly pinning to oldest support versions of dependencies)
 rapids-generate-pip-constraints py_test_dask_cudf ./constraints.txt
 
+# latest-nightly builds are run against upstream *dev* versions of various packages:
+if [[ "${RAPIDS_DEPENDENCIES}" == "latest" ]] && [[ "${BUILD_TYPE}" == "nightly" ]]; then
```

Comment:
`RAPIDS_DEPENDENCIES` is set [here](https://github.com/rapidsai/shared-workflows/blob/c6d27a0215545cc0ce0216d794ec114ee5fe215d/.github/workflows/wheels-test.yaml#L112) and
`BUILD_TYPE` is set [here](https://github.com/rapidsai/shared-workflows/blob/c6d27a0215545cc0ce0216d794ec114ee5fe215d/.github/workflows/wheels-test.yaml#L61C7-L61C17)

With `RAPIDS_DEPENDENCIES="latest"`, rapids-generate-pip-constraints currently outputs an empty text file: https://github.com/rapidsai/gha-tools/blob/4e9add6b630887217825bbbcd2009742dab271ec/tools/rapids-generate-pip-constraints#L32-L34.